api/apps/geoprocessing/src/modules/planning-area/marxan-geo-json.ts

```diff
@@ -0,0 +1,32 @@
+import { BBox, GeoJSON } from 'geojson';
+import { PuSizes } from './pu-sizes';
+
+export class MarxanGeoJson {
+  private readonly geoJson: GeoJSON & {
+    marxanMetadata?: Record<string, unknown>;
+  };
+
+  constructor(geoJson: GeoJSON) {
+    this.geoJson = geoJson;
+  }
+
+  addBBox(bbox: BBox) {
+    this.geoJson.bbox = bbox;
+  }
+
+  addPuSizes(puSizes: PuSizes) {
+    this.addMarxanMetadata('maxPuAreaSize', puSizes.maxPuAreaSize);
+    this.addMarxanMetadata('minPuAreaSize', puSizes.minPuAreaSize);
+  }
```

Comment:
that's actually `set` but its less than minor

Reply:
I've removed these methods